controllers/managedocs_controller.go

```diff
@@ -405,6 +407,16 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 
 	if !r.managedOCS.DeletionTimestamp.IsZero() {
 		if r.verifyComponentsDoNotExist() {
+			// Deleting OCS CSV from the namespace
+			r.Log.Info("deleting OCS CSV")
+			csv, err := r.getCSVByPrefix(ocsOperatorName)
+			if err != nil {
+				return ctrl.Result{}, err
+			}
+			if err := r.delete(csv); err != nil {
+				return ctrl.Result{}, fmt.Errorf("unable to delete csv: %v", err)
+			}
```

Comment:
Lets not pollute the function scope with an err var

```suggestion
			if csv, err := r.getCSVByPrefix(ocsOperatorName); err == nil {
				if err := r.delete(csv); err != nil {
					return ctrl.Result{}, fmt.Errorf("unable to delete csv: %v", err)
				}
			} else {
				return ctrl.Result{}, err
			}		
```